config/jobs/image-pushing/k8s-staging-kind.yaml

```diff
@@ -0,0 +1,27 @@
+postsubmits:
+  kubernetes-sigs/kind:
+    - name: post-kind-push-binaries
+      cluster: test-infra-trusted
+      annotations:
+        testgrid-dashboards: sig-testing-kind
+        testgrid-alert-email: kubernetes-engprod+alerts@google.com,antonio.ojea.garcia@gmail.com
+        testgrid-num-columns-recent: '3'
+      decorate: true
+      spec:
+        serviceAccountName: deployer # TODO(fejta): use pusher
+        containers:
+          - image: gcr.io/k8s-testimages/image-builder:v20200213-0032cdb
+            command:
+              - /run.sh
+            args:
+              - --project=k8s-staging-kind
+              - --scratch-bucket=gs://k8s-infra-staging-kind-gcb
+              - --env-passthrough=PULL_BASE_REF
+              - --build-dir=.
+              - hack/ci/push-latest-cli/
+            env:
+              - name: LOG_TO_STDOUT
+                value: "y"
+      rerun_auth_config:
+        github_team_ids:
+          - 2921947 # kind-maintainers
```

Comment:
instead of mucking about with the API you can find these using the avatar source in the team page


e.g. release managers: https://github.com/orgs/kubernetes/teams/release-managers
=> https://avatars0.githubusercontent.com/t/2241179?s=280&v=4

https://github.com/orgs/kubernetes-sigs/teams/kind-maintainers
=> https://avatars0.githubusercontent.com/t/2921947?s=280&v=4